src/Umbraco.Web/PropertyEditors/BlockListConfiguration.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Newtonsoft.Json;
+using Umbraco.Core.PropertyEditors;
+
+namespace Umbraco.Web.PropertyEditors
+{
+
+    /// <summary>
+    /// The configuration object for the Block List editor
+    /// </summary>
+    public class BlockListConfiguration
+    {
+        [ConfigurationField("elementTypes", "Element Types", "views/propertyeditors/blocklist/blocklist.elementtypepicker.html", Description = "Select the Element Types to use as models for the items.")]
+        public ElementType[] ElementTypes { get; set; }
+
+        // TODO: Fill me in
```

Comment:
Is this a leftover TODO @Shazwazza or this for a next phase/iteration of work once synced up with Niels L or is something important missing here?!

Reply:
This is for the next phase of work to be done, we haven't created the property editor yet at thsi stage.